services/bookService.js

```diff
@@ -0,0 +1,20 @@
+const bookDao = require('../models/bookDao');
+
+const getBookById = async (bookId) => {
+  if (!(await isExistedBook(bookId))) {
+    const error = new Error('BOOK_DOES_NOT_EXIST');
+    error.statusCode = 400;
+    throw error;
+  }
+
+  return bookDao.getBookById(bookId);
+};
+
+const isExistedBook = async (bookId) => {
+  return bookDao.isExistedBook(bookId);
+};
```

Comment:
제가 왜 이렇게 바꿨는지 잘 고민해 보시고 고민한 내용을 PR template에 남겨주세요.
```suggestion
  const book = await bookDao.getBookById(bookId);
  
  if (!book) {
    const error = new Error('BOOK_DOES_NOT_EXIST');
    error.statusCode = 404;
    throw error;
  }

  return book
};

```